site/styles/_variables.scss

```diff
@@ -37,9 +37,9 @@
   --page-padding-top: 120px;
   --page-padding-top-md: 120px;
   --page-padding-top-lg: 180px;
-  --page-detail-padding-top: 120px;
-  --page-detail-padding-top-md: 120px;
-  --page-detail-padding-top-lg: 180px;
+  --page-detail-padding-top: 245px;
+  --page-detail-padding-top-md: 290px;
+  --page-detail-padding-top-lg: 360px;
```

Comment:
@curtisadams28 does this have any impact on what you are currently working on?

@matthewthomsonnz have you tested what changing these values does to every page that uses them (film/bundle/tv season)?

Reply:
Should be fine

Reply:
I had checked film/tv were consistent but not bundle.

The spacing between top of content and nav looks good at all breakpoints. Matches tv and film.